LR_Modules/setup_nscf.f90

```diff
@@ -151,6 +153,20 @@ SUBROUTINE setup_nscf ( newgrid, xq, elph_mat )
           npk, k1, k2, k3, nk1, nk2, nk3, nkstot, xk, kunit)
   END IF
   !
+#if defined(__SIRIUS)
```

Comment:
This is a copy-paste of the code from setup.f90. Better to move it to a standalone function.